examples/gallery/images/cross_section.py

```diff
@@ -1,6 +1,7 @@
 """
 Cross-section along a transect
-==============================
+------------------------------
```

Comment:
Revert this change and address it in a separate PR?
```suggestion
==============================
```